external/netfx/netfx.depproj

```diff
@@ -25,5 +25,11 @@
     <FileToExclude Include="System.EnterpriseServices.Thunk" />
     <FileToExclude Include="System.EnterpriseServices.Wrapper" />
   </ItemGroup>
+  <ItemGroup Condition="'$(TargetGroup)'=='uap' or '$(TargetGroup)'=='uapaot'">
```

Comment:
Does this really belong to this file? It does not sound right to have coupling between netfx and uap/uapaot.